src/rollback_to_stable/rts_api.c

```diff
@@ -138,6 +140,11 @@ __rollback_to_stable_one(WT_SESSION_IMPL *session, const char *uri, bool *skipp)
 
     WT_RET(__wt_metadata_search(session, uri, &config));
 
+    /* Initialize the tracking timer. */
```

Comment:
It might be worth moving this up before the metadata search, in case that's (somehow) slow.

Reply:
Fixed.